routes/orders.js

```diff
@@ -16,6 +16,7 @@ router.get('/', async (req, res) => {
   } else {
     res.sendStatus(404)
   }
+  return orders
```

Comment:
I should have been more clear, I suggest returning from the function in the `catch` block, so that the rest of the computation doesn't happen. Returning `orders` here does nothing, since this is the last function in the middleware chain (since you haven't called `next()` anywhere).